examples/artifacts/plot_correct_ajdc_EEG.py

```diff
@@ -0,0 +1,163 @@
+"""
+=========================================================
+Artifact Correction by AJDC-based Blind Source Separation
+=========================================================
+
+Blind source separation (BSS) based on approximate joint diagonalization of 
+Fourier cospectra (AJDC), applied to artifact correction of EEG [1].
+"""
+# Authors: Quentin Barthélemy & David Ojeda.
+# EEG signal kindly shared by Marco Congedo.
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from mne import create_info
+from mne.io import RawArray
+from mne.viz import plot_topomap
+from mne.time_frequency import psd_welch
+from mne.preprocessing import ICA
+from pyriemann.spatialfilters import AJDC
+from matplotlib import pyplot as plt
+
+import warnings
+from mne import __version__ as v
+if v != '0.18.2':
+    warnings.warn('This example has only been tested with mne version 0.18.2',
+                  UserWarning)
```

Comment:
you can take this commit to avoid this:

https://github.com/agramfort/pyRiemann/commit/927459c0958feb95ddaf4e471ca49b6e1f21ca3c

Reply:
Thank you Alexandre! I am going to upgrade mne and sklearn.

Reply:
Upgrading mne from 0.18 to 0.21, it seems that channel positions have changed when displaying the topographic map of the blink source, using `plot_topomap()`. 

MNE 0.18
![blink_topomap_mne018](https://user-images.githubusercontent.com/5346669/97625865-5d865c80-1a29-11eb-9512-5b23d5ec27eb.png)

MNE 0.21
![blink_topomap_mne021](https://user-images.githubusercontent.com/5346669/97625899-6bd47880-1a29-11eb-90b5-7a56ae20ea14.png)


The difference is significant for frontal channels: Fp1, Fp2, F7, F3, F4, F8.
How can I fix that? @agramfort

Reply:
I don't have enough background to tell what the problem could be. What exactly is different between MNE 0.18 and 0.21? The topomaps? I assume it's not just the visualization that's different (because (1) we changed the spherical projection at some point and (2) the sign is irrelevant). Which MNE functions does your method use that could have caused the result to change?

Reply:
@qbarthelemy Try using the `sphere` argument - the default sphere radius is `0.095` (in meters, so that's 9.5 cm) which may not be adequate for your channel positions. Take a look at the tutorial [here](https://mne.tools/dev/auto_tutorials/intro/plot_40_sensor_locations.html#controlling-channel-projection-mne-vs-eeglab) and [this example](https://mne.tools/dev/auto_examples/visualization/plot_eeglab_head_sphere.html#sphx-glr-auto-examples-visualization-plot-eeglab-head-sphere-py).

Reply:
Thank you for your quick help!

@cbrnr Yes I agree: due to BSS uncertainty on sign and order of sources, the sign is not a problem. My question is about channel locations: in MNE 0.21, Fp1 and Fp2 are right next to the nose, and F7, F3, F4, F8 are curved downward. This is different from MNE 0.18.
I have updated my comment above, to highlight the differences between my example coded with MNE 0.18, and then with MNE 0.21.

 @mmagnuski Thanks for the links! I have read the tutorial several times, but I can't properly set the values of  `sphere` argument in MNE 0.21, because the documentation of MNE 0.18 does not state this information. So, I try to manually tune these values in order to reproduce the locations of MNE 0.18. But, I don't find (yet) the configuration ensuring backward compatibility with previous versions of MNE.

Reply:
When you say you try to reproduce the original locations, are you measuring this by eye or do you have 3D coordinates for 0.18 and 0.21?

Reply:
@qbarthelemy The locations are actually the same, they are just differently projected to 2d with reference to a sphere. If you don't like how the channels are projected by default in 0.21 - now you have full control to specify the origin and radius of the sphere  used in the 3d -> 2d projection. To get exactly what you saw in 0.18 might take a bit of trial and error, but in general try moving the sphere origin to the front and reduce the sphere radius.

Reply:
My approach was a simple try-and-error optimization, with a subjective visual evaluation. I have not reached the solution, in spite of several iterations. So, this "à la hussarde" strategy does not seem appropriate to converge.
It probably exists a closed-form solution to convert this 3D to 2D projection from MNE 0.18 to 0.21, requiring to dig into linear algebra.

But on reflection, this topomap is not the crucial part of my example, so, I have decided to keep the current display. Thanks for all!

Reply:
@qbarthelemy No problem. In your specific case it might indeed be impossible to get exactly the same projection as in 0.18. First, 0.18 had a different srategy for topo channel placement - first the channels were projected to a sphere and then they would be stretched until they filled the whole head circle. Now the channels are not projected to the sphere surface, only with respect to its position and radius. Then, because `'standard_1020'` used by mne currently contains realistic, not schematic, channel positions. Because these channels are not projected to the sphere surface now, their constellation may appear more eliptical - but that is due to the shape of the standard_1020 positions used by mne. This may be resolved one day though: mne-tools/mne-python#7141 :)